angr/block.py

```diff
@@ -248,7 +248,7 @@ def __ne__(self, other):
 
     def pp(self, **kwargs):
         if self._project is not None:
-            print(self._project.analyses.Disassembly(ranges=[(self.addr, self.addr + self.size)]).render(**kwargs))
+            print(self._project.analyses.Disassembly(ranges=[(self.addr - self.thumb, self.addr - self.thumb + self.size)]).render(**kwargs))
```

Comment:
Disassembly analysis can support thumb node, but instead of just changing the address here let's pass `thumb` as a new parameter to `Disassembly` and then add that to the `BlockNode`'s that get created.